src/CMakeLists.txt

```diff
@@ -777,6 +777,13 @@ else(USE_GAME)
   message(STATUS "Game: no")
 endif(USE_GAME)
 
+if(USE_SIGNAL_TRACE)
```

Comment:
Why not enable support for this unconditionaly ? As for all debug option we just need "-d <name>" to enable it. Maybe there is something special on this one?

Reply:
It's very noisy tracing option that prints call backtrace, which would be only useful in very marginal situations like "from where the hell this signal raise/connect/disconnect came???" If enabled it slows down things by a bit due to checking for backtrace. This tracing makes sense imo only for devs or very very narrow use case and also requires GCC because it uses gnu extensions. So it IMO should be behind an option and hidden by default :)

Reply:
Sure it is very noisy like "-d sql" my point is not to make it always on but to enable it only at runtime when needed with "-d signal" and so I don't see the point of having this enabled/disabled at config time in cmake.

Reply:
how about now? if you compiled darktable with gcc, it defines that it's possible to print trace and you get `--d-signal-act print-trace` option to print trace. it's additionally conditional so I think it's low impact.

Reply:
Looks better to me! Thanks.